bin/git-ensembl

```diff
@@ -609,7 +609,7 @@ sub _default_groups {
       },
       production    => {
           desc    => 'Libraries required to run Ensembl production',
-          modules => [ qw/ensembl ensembl-biomart ensembl-compara ensembl-database-loader ensembl-datacheck ensembl-funcgen ensembl-hive ensembl-io ensembl-metadata ensembl-ontology-schema ensembl-orm ensembl-prodinf-core ensembl-prodinf-srv ensembl-production ensembl-taxonomy ensembl-tools ensembl-variation VersioningService ensembl-vep ols-client ols-ensembl-loader/ ],
+          modules => [ qw/ensembl ensembl-biomart ensembl-compara ensembl-datacheck ensembl-funcgen ensembl-hive ensembl-io ensembl-metadata ensembl-ontology-schema ensembl-orm ensembl-prodinf-core ensembl-prodinf-tools ensembl-production ensembl-taxonomy ensembl-tools ensembl-variation ensembl-vep ols-client ols-ensembl-loader/ ],
```

Comment:
I don't believe we need to actually have those repos checked out:
> ensembl-prodinf-core
> ensembl-prodinf-tools 
The former is no use for any pipeline I am aware of, and should be installed as a dependency in a virtual env
The latter is supposed to be installed via pip, then the `setup.py` will do the trick to add the commands in prompt. 
Since services are now run from checkout of ensembl-production-configuration repo, that should be ok.

Reply:
My thinking was that we sometimes call the clients as part of the production processes (e.g. https://www.ebi.ac.uk/seqdb/confluence/display/GTI/Updating+the+metadata+database#Updatingthemetadatadatabase-Execution). But I'm not quite sure if I have wrapped my head around how things are going to work, so if you think it best, I can remove them.

Reply:
you make me doubt now :-) From the example mentioned, this is typically the use case for one "install" of prodinf-tools in a virtualenv 
So the SOP would change to something like:
```
$: pyenv activate prodinf-tools
$(prodinf-tools): metadata_client.py ....
```
I can't see any use case where we shouldn't use a dedicated virtual env with those tow libraries installed.

Reply:
Agreed, I get it now, thanks. Repos removed from the group.